cmd/intercept/intercept.go

```diff
@@ -125,6 +127,8 @@ func main() {
 				msgAccession: "accessionIDs",
 				msgIngest:    "ingest",
 				msgMapping:   "mappings",
+				msgRelease:   "mappings",
+				msgDeprecate: "mappings",
```

Comment:
`msgRelease` and `msgDeprecate` should probably be something different than `mappings`, but not sure what, since there are similar things many times (singular, plural, prefix-name).

If I understand correctly, we should use these constants https://github.com/neicnordic/sda-pipeline/blob/d4eefc99c44652efe1ea61bde9ae95a5012de496/cmd/intercept/intercept.go#L16-L23 instead of building the dictionary here https://github.com/neicnordic/sda-pipeline/blob/d4eefc99c44652efe1ea61bde9ae95a5012de496/cmd/intercept/intercept.go#L126-L132 and here https://github.com/neicnordic/sda-pipeline/blob/d4eefc99c44652efe1ea61bde9ae95a5012de496/cmd/intercept/intercept.go#L161-L168